pkg/relayer/session/sessiontree.go

```diff
@@ -72,14 +72,18 @@ func NewSessionTree(
 ) (relayer.SessionTree, error) {
 	// Join the storePrefix and the session.sessionId and supplier's operator address to
 	// create a unique storePath.
-	storePath := filepath.Join(storesDirectory, sessionHeader.SessionId, "_", supplierOperatorAddress.String())
+
+	// TODO_IMPROVE: instead of creating a new KV store for each session, it will be more beneficial to
```

Comment:
Nice call out! :fire: 

This seems like something we should create an issue for it one does not already exist (and reference here).

Reply:
@bryanchriswhite, there's #621 where we track current performance issues. Honestly, I don't see us focusing on low-level, deep performance optimizations at this stage. By the time we need them, the codebase might be in a completely different place.

👍 or 👎 ?

Reply:
I was only suggesting referencing the issue number like so:

```suggestion
	// TODO_IMPROVE(#621): instead of creating a new KV store for each session, it will be more beneficial to
```

Reply:
I was only suggesting referencing the issue number like so:

```suggestion
	// TODO_IMPROVE(#621): instead of creating a new KV store for each session, it will be more beneficial to
```